bottleneck.d.ts.ejs

```diff
@@ -0,0 +1,176 @@
+
+declare module "bottleneck" {
+    namespace Bottleneck {
+        type Callback<T> = (err: any, result: T) => void;
```

Comment:
The callback can return more than one result, but I understand that having to define
`type Callback2<T,U> = (err: any, result1: T, result2: U) => void;` and then `Callback3` and so on would make the whole thing exponentially complicated since `submit()` and `submitPriority()` are declared in a loop. If you think of a simple way to do it, please feel free to implement it :)